### PR TITLE
Add SFI scheme codes

### PR DESCRIPTION
**Pull request by johnwatson484**

### CODE REVIEW

File: app/schedules.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  QUARTERLY: 'Q4',
+  MONTHLY: 'M12',
+  THREE_DAY_QUARTERLY: 'T4'
+}
```
Comment (marctemp) [inline app/schedules.js:5]: BIG fan of this.

Only issue I have is we have in our other services constant schedules but their names are simply `Q4`, `M12` and `T4`.

For consistency sake I would suggest changing these to those already defined.

HOWEVER, I propose that these constant names are ssooooo much better than merely the DAX schedule name and so I would suggest adding a tech debt ticket to update the constant schedules to these across all of our services.

File: test/integration/local/inbound/save-payment-request.test.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const db = require('../../../../app/data')
 const savePaymentRequest = require('../../../../app/inbound')
+const { MONTHLY } = require('../../../../app/schedules')
 let scheme
 let paymentRequest
 describe('save payment requests', () => {
```
Comment (marctemp) [inline test/integration/local/inbound/save-payment-request.test.js:6]: Consider adding whitespace

File: app/plugins/logging.js [hunk 3]
```diff
@@ -1,10 +1,7 @@
-const config = require('../config')
-
 module.exports = {
   plugin: require('hapi-pino'),
   options: {
     logPayload: true,
-    prettyPrint: config.isDev,
```
Comment (marctemp) [inline app/plugins/logging.js:7]: What's the reason for removing this?

I can see this being used in the following repos:
- `ffc-pay-web`
- `ffc-pay-processing`
- `ffc-pay-request-editor`
- `azure-service-bus-message-hospital`

All of which bar the last have this configured.

**Review by marctemp — CHANGES_REQUESTED** [review]
Just some minor changes, suggestions and comments.

1 extra which I cnt comment on as the file hasn't been changed:

- in the test files: `test/integration/local/processing/apply-auto-hold.test.js`, `test/integration/local/processing/map-account-codes.test.js` and `test/integration/local/processing/process-payment-requests.test.js`
- they make reference to insert into the `accountCode` tbl for `G00 - Gross value of claim` and `P24 - Over declaration reduction` for `schemeId` 1, i.e. `SFI`
- however, the `accountCodeAP`, `accountCodeARIrr` and `accountCodeARAdm` are wrong
- could you update the values (I appreciate it doesn't have any influence over those tests but for mimicking real life values as close as possible they should be accurate

File: app/schedules.js (hunk 1 above)
Comment (johnwatson484) [reply app/schedules.js:5]: I did originally add them as `Q4`, `M12` and `T4` but felt like you they aren't that clear what they mean.  `Q4` and `M12` have some logic to them, but `T4` and perhaps others that we add don't.

But agree consistency is important, so I've changed them back.

Tech debt ticket created to update all of them to more meaningful names -> https://eaflood.atlassian.net/browse/SFI-2153

File: test/integration/local/inbound/save-payment-request.test.js (hunk 2 above)
Comment (johnwatson484) [reply test/integration/local/inbound/save-payment-request.test.js:7]: Considered and done.

File: app/plugins/logging.js (hunk 3 above)
Comment (johnwatson484) [reply app/plugins/logging.js:7]: It no longer exists in the updated version of `hapi-pino`.

There is a separate package that needs to be installed and configured to get _similar_ capability, [`pino-pretty`](https://github.com/pinojs/pino-pretty)

The pretty only kicks in for local dev and we removed it from all the SFI services without issue.  Updated Pino logs a bit neater than older ones so our original need for it is gone so didn't see the value in adding another dependency.

**Comment by johnwatson484** [comment]
> Just some minor changes, suggestions and comments.
> 
> 1 extra which I cnt comment on as the file hasn't been changed:
> 
> * in the test files: `test/integration/local/processing/apply-auto-hold.test.js`, `test/integration/local/processing/map-account-codes.test.js` and `test/integration/local/processing/process-payment-requests.test.js`
> * they make reference to insert into the `accountCode` tbl for `G00 - Gross value of claim` and `P24 - Over declaration reduction` for `schemeId` 1, i.e. `SFI`
> * however, the `accountCodeAP`, `accountCodeARIrr` and `accountCodeARAdm` are wrong
> * could you update the values?
> * I appreciate it doesn't have any influence over those tests but for mimicking real life values as close as possible they should be accurate

Fair comment.  Updated.

File: test/integration/local/inbound/save-payment-request.test.js [hunk 4]
```diff
@@ -82,13 +82,13 @@ describe('save payment requests', () => {
     })
 
     expect(invoiceLinesRows[0].schemeCode).toBe('80001')
-    expect(invoiceLinesRows[0].accountCode).toBe('SOS273')
+    expect(invoiceLinesRows[0].accountCode).toBe('SOS710')
```
Comment (marctemp) [inline test/integration/local/inbound/save-payment-request.test.js:85]: aaahhh we do check -- marvellous 🔥

File: app/plugins/logging.js (hunk 3 above)
Comment (marctemp) [reply app/plugins/logging.js:7]: Agree -- it sounds like we should remove this from the other repos that use it too?

Can you create a tech debt ticket to do so?

Comment (marctemp) [reply app/plugins/logging.js:7]: Agree -- it sounds like we should remove this from the other repos that use it too?

Can you create a tech debt ticket to do so?

**Review by marctemp — APPROVED** [review]
1 tech debt ticket to be created re: removing `prettyPrint` in other repos which use these as per your comment -- other than that, all groovy

**Pull request merged**